app/models/article.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: articles
+#
+#  id           :bigint           not null, primary key
+#  for_banks    :boolean
+#  for_partners :boolean
+#  question     :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+class Article < ApplicationRecord
```

Comment:
Is Article really the right name for this model? I think something like Question or even FAQ might be more obvious as to what it does.

Reply:
I think that FAQ sounds good. The only reason calling the model Question sounds less ideal to me is that the model already has a column called question.